modules/ROOT/pages/apikit-asyncapi-module-reference.adoc

```diff
@@ -0,0 +1,200 @@
+| Kafka Message Broker | Supports asynchronous messaging with Apache Kafka. | kafka, kafka-secure | Kafka configurations are specified in the server object of the AsyncAPI specification. Supports both publish and subscribe operations. 
+|===
+
+==== Associated Operations
+* <<Publish>>  
+
+==== Associated Sources
+* <<MessageListener>>  
+
+== Operations
+
+[[Publish]]
+== Publish
+`<apikit-asyncapi:publish>`
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use | None | *Yes* 
+| Content | Binary | None |  #[payload] | No  
+| Output MIME Type | String | MIME type of the payload that this operation outputs | None | No 
+| Output Encoding | String | Encoding of the payload that this operation outputs | None | No 
+| Config Ref | ConfigurationProvider | Name of the configuration used to execute this component | None | *Yes* 
+| Channel Name | String | Channel name | None | *Yes* 
+| Server Name | String | Server name | None | *Yes* 
+| Protocol Specific Parameters a| One of:
+
+* <<KafkaPublishParameters>> 
+
+| Protocol-specific parameters | None | No  
+| Target Variable | String | Name of a variable on which the operation's output is placed | None | No 
+| Target Value | String | Expression that is evaluated against the operation's output and the effect of that expression is stored in the target variable | #[payload] | No 
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+| *Type* | Any
+| *Attributes Type* | Any
+|===
+
+=== For Configurations
+
+* <<Config>>  
+
+=== Throws
+
+* APIKIT-ASYNCAPI:ASYNCAPI_KIT_INTERNAL_SERVER_ERROR  
+
+== Sources
+
+[[MessageListener]]
+== Message Listener
+`<apikit-asyncapi:message-listener>`
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use | None | *Yes* 
+| Channel Name | String | Channel name | None | *Yes* 
+| Servers | Array of String | List of servers to listen to | None | No 
+| Output MIME Type | String | MIME type of the payload that this operation outputs | None | No 
+| Output Encoding | String | Encoding of the payload that this operation outputs | None | No  
+| Primary Node Only | Boolean | Specifies if this source must be executed on the primary node when running in Cluster | None | No 
+| Config Ref | ConfigurationProvider | Name of the configuration used to execute this component | None | *Yes* 
+| Redelivery Policy | <<RedeliveryPolicy>> | Policy for processing the redelivery of the same message | None | No
+| Body Content | Any | Content of the message body | #[payload] | No
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+| *Type* | Any
+| *Attributes Type* | AsyncMessageAttributes
+|===
```

Comment:
Attributes type is One of Kafka Message Attributes or Anypoint MQ Message Attributes